angular-translate/angular-translate.d.ts

```diff
@@ -72,7 +72,7 @@ declare namespace angular.translate {
     }
 
     interface ITranslateProvider extends angular.IServiceProvider {
-        translations(): ITranslationTable;
+        translations(key?: string): ITranslationTable;
         translations(key: string, translationTable: ITranslationTable): ITranslateProvider;
```

Comment:
actually I was trying to cover https://github.com/angular-translate/angular-translate/blob/master/src/service/translate.js#L243

and i thought that return type for it would be same as for https://github.com/angular-translate/angular-translate/blob/master/src/service/translate.js#L238 thats why `ITranslationTable` was used.. However I may be wrong